operations/fetch_test.go

```diff
@@ -124,7 +124,6 @@ func TestFileNameWithIndex(t *testing.T) {
 }
 
 func TestResetGitRemoteToSSH(t *testing.T) {
-
```

Comment:
> it looks like the token that was leaked was separate from the task that it somehow picked up from somewhere else- so the task could have known to leak it. Our redaction logic doesn't extend to tokens created in tests.

I could see a ticket for adding redaction to tests but it wouldn't have prevented this and we don't have that many e2e tests. 

Let's add this context to the ticket that we created for it.

Reply:
I have some further thoughts that I sent over slack